include/tcpdf_static.php

```diff
@@ -1887,7 +1887,8 @@ public static function encodeUrlQuery($url) {
 		if (isset($urlData['query']) && $urlData['query']) {
 			$urlQueryData = array();
 			parse_str(urldecode($urlData['query']), $urlQueryData);
-			$updatedUrl = $urlData['scheme'] . '://' . $urlData['host'] . $urlData['path'] . '?' . http_build_query($urlQueryData);
+			$port = isset($urlData['port']) ? ':'.$urlData['port'] : '';
+			$updatedUrl = $urlData['scheme'].'://'.$urlData['host'].$port.$urlData['path'].'?'.http_build_query($urlQueryData);
```

Comment:
```suggestion
			$port = isset($urlData['port']) ? ':' . $urlData['port'] : '';
			$updatedUrl = $urlData['scheme'] . '://' . $urlData['host'] . $port . $urlData['path'] . '?' . http_build_query($urlQueryData);
```

Reply:
tcpdf seems to concatenate strings without whitespace.

Reply:
Oh okay, this is terrible to read but okay ^^